package.json

```diff
@@ -169,6 +170,7 @@
     "@octokit/rest": "^18.12.0",
     "@styled-system/prop-types": "^5.1.5",
     "@tanstack/react-virtual": "^3.0.0-beta.39",
+    "@types/jquery": "^3.5.16",
```

Comment:
Does this need to be dependency? Is it used by anything that goes into production?

Reply:
removed